packages/block-library/src/navigation/edit.js

```diff
@@ -84,7 +77,7 @@ function Navigation( {
 			className: 'wp-block-navigation__container',
 		},
 		{
-			allowedBlocks: ALLOWED_BLOCKS,
+			allowedBlocks: allowBlocksList,
```

Comment:
This looks good I think. Why not have the prop be named simply `allowedBlocks` just as the innerBlocks api expects?